doc/neps/nep-0049.rst

```diff
@@ -153,12 +156,26 @@ NumPy C-API functions
    hooks. All the function pointers must be filled in, ``NULL`` is not
    accepted.
 
-.. c:function:: const char * PyDataMem_GetHandlerName(PyArrayObject *obj)
+.. c:function:: const PyDataMem_Handler * PyDataMem_GetHandler(PyArrayObject *obj)
 
-   Return the const char name of the ``PyDataMem_Handler`` used by the
+   Return the ``PyDataMem_Handler`` used by the
    ``PyArrayObject``. If ``NULL``, return the name of the current global policy
    that will be used to allocate data for the next ``PyArrayObject``.
```

Comment:
I don't think "return the name" is correct any more.

Reply:
fixed